app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -618,16 +615,35 @@ public boolean onTouch(View v, MotionEvent event) {
      */
     private void createInfoLayout() {
 
-        LayoutInflater inflater = (LayoutInflater) getSystemService(LAYOUT_INFLATER_SERVICE);
-        infoLayout = (LinearLayout) inflater.inflate(R.layout.dialog_info_window, null);
-        layoutParams.gravity = Gravity.CENTER | Gravity.BOTTOM;
```

Comment:
I intend to clean up that too. If it's in the class it should be with the other declarations for layoutParams (line 268). At the moment it's also declared on the XML files themselves, I should get rid of that and place everything together in one place.

It doesn't make sense adjusting parameters (that should be default) here.